x-pack/plugins/security_solution/public/overview/components/detection_response/alerts_by_status/alerts_by_status.tsx

```diff
@@ -177,55 +192,88 @@ export const AlertsByStatus = ({
           {toggleStatus && (
             <>
               <EuiFlexGroup justifyContent="center" gutterSize="none">
-                <EuiFlexItem grow={false}>
-                  {totalAlerts !== 0 && (
-                    <EuiText className="eui-textCenter" size="s">
-                      <>
-                        <b>
-                          <FormattedCount count={totalAlerts} />
-                        </b>
-                        <> </>
-                        <small>{ALERTS(totalAlerts)}</small>
-                      </>
-                    </EuiText>
-                  )}
+                <EuiFlexItem grow={isChartEmbeddablesEnabled}>
+                  {totalAlerts !== 0 ||
+                    (visualizationTotalAlerts !== 0 && (
+                      <EuiText className="eui-textCenter" size="s">
+                        <>
+                          <b>
+                            <FormattedCount count={totalAlertsCount} />
+                          </b>
+                          <> </>
+                          <small>{ALERTS(totalAlertsCount)}</small>
+                        </>
+                      </EuiText>
+                    ))}
                   <EuiSpacer size="l" />
                   <EuiFlexGroup justifyContent="center">
-                    <StyledFlexItem key="alerts-status-open" grow={false}>
-                      <DonutChart
-                        data={donutData?.open?.severities}
-                        fillColor={fillColor}
-                        height={donutHeight}
-                        label={STATUS_OPEN}
-                        title={<ChartLabel count={openCount} />}
-                        totalCount={openCount}
-                      />
+                    <StyledFlexItem key="alerts-status-open" grow={isChartEmbeddablesEnabled}>
+                      {isChartEmbeddablesEnabled ? (
+                        <AlertDonutEmbeddable
+                          status="open"
+                          timerange={timerange}
+                          label={STATUS_OPEN}
+                        />
+                      ) : (
+                        <DonutChart
+                          data={donutData?.open?.severities}
+                          fillColor={fillColor}
+                          height={donutHeight}
+                          label={STATUS_OPEN}
+                          title={<ChartLabel count={openCount} />}
+                          totalCount={openCount}
+                          isChartEmbeddablesEnabled={isChartEmbeddablesEnabled}
```

Comment:
if `isChartEmbeddablesEnabled` is true though, this component wouldnt render as As AlertDonutEmbeddable would :P. Same for the instances below it. I think you can remove the prop.

Reply:
OK, I have this removed in this PR: https://github.com/elastic/kibana/pull/147261/files